src/main/drivers/system_stm32f7xx.c

```diff
@@ -29,11 +29,37 @@
 #include "drivers/exti.h"
 #include "drivers/nvic.h"
 #include "drivers/system.h"
+#include "drivers/light_led.h"
+#include "drivers/time.h"
+#include "drivers/transponder_ir.h"
 
 
 #define AIRCR_VECTKEY_MASK    ((uint32_t)0x05FA0000)
 void SystemClock_Config(void);
 
+void MemManage_Handler(void)
+{
+    LED2_ON;
+
+#ifdef USE_TRANSPONDER
+    // prevent IR LEDs from burning out.
+    uint8_t requiredStateForTransponder = SYSTEM_STATE_CONFIG_LOADED | SYSTEM_STATE_TRANSPONDER_ENABLED;
```

Comment:
This won't work, because these are defined in `fc_init.h`. And we should not include this in the driver layer. Can we just force disable the transponder?

Reply:
I think of moving it into `fc_hardfaults.c`

Reply:
That sounds like a good approach to get out of this.